src/backend/distributed/transaction/backend_data.c

```diff
@@ -998,6 +999,19 @@ citus_backend_gpid(PG_FUNCTION_ARGS)
 }
 
 
+/*
+ * citus_backend_nodeid similar to pg_backend_pid, but returns Citus
```

Comment:
There is not really a "backend node id" concept, isn't is just `citus_get_nodeid()`? I mean, all backends have the same nodeid.

Well also super minor: "assigned nodeid" is right, we assign it, but only once in the lifetime of a node. So, maybe just drop "assigned"?

Reply:
Removed function again.